test/box-luatest/access_control_test.lua

```diff
@@ -0,0 +1,280 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_each(function(cg)
+    cg.server = server:new()
+    cg.server:start()
+end)
+
+g.after_each(function(cg)
+    cg.server:drop()
+end)
+
+-- Make sure requested privileges granted properly.
+g.test_grant = function(cg)
+    cg.server:exec(function()
+        local luatest_helpers = require('luatest.helpers')
+        local access = require('access_control')
```

Comment:
Nit:
Sorry for being such a nerd, but can you use `access_control` or `ac` instead of just `access`?